client/components/Recipe/Recipe.tsx

```diff
@@ -20,6 +20,10 @@ const Recipe = ({ post }: { post: IPost }) => {
     formState: { errors, isSubmitting },
   } = useForm();
 
+  const [imageSrc, setImageSrc] = useState(
+    `https://food-blog-server1.onrender.com/api${post.photo.imageLink}`
```

Comment:
this is an example of where having a hard coded API URL is helpful. if u ever need to change that string you can do it in one place.